data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/Main.java

```diff
@@ -64,9 +66,11 @@ public static void main(final String[] args) throws IOException {
 
     // Read consumer and producer kafka config
     Properties producerConfig = Configurations.readPropertiesSync(env.getProducerConfigFilePath());
+    producerConfig.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
```

Comment:
why did we have this before? B/c was just string - no SerDe needed?

Reply:
We got this config from the CM with all producer configurations but now I want to force it to be `String` so that no one can mess with it.